app/src/main/java/org/simple/clinic/drugs/EditMedicinesUpdate.kt

```diff
@@ -2,6 +2,7 @@ package org.simple.clinic.drugs
 
```

Comment:
Since this is business logic, can we reword the commit message so that it is not implementation specific?

Suggestion: "Set button state to 'Refill Medicine' when the prescription has been changed on the current date".